packages/connect-migrate/src/transforms/__testfixtures__/modify-imports/simple.input.ts

```diff
@@ -0,0 +1,4 @@
+import { createPromiseClient } from "@bufbuild/connect";
```

Comment:
Single quotes will be very common. Are they preserved?

Reply:
Just pushed some code up to detect the style used (based on the first import detected) and we maintain that style.